exotic/inputs.py

```diff
@@ -507,8 +507,9 @@ def comparison_star_coords(comp_stars, rt_bool):
             all(isinstance(star, list) for star in comp_stars):
         comp_stars = [star for star in comp_stars if star != []]
     elif isinstance(comp_stars, str) and any(str.isdigit(x) for x in comp_stars):
-        comp_stars = ' '.join(re.sub('\D', ' ', comp_stars).split()).split(' ')
-        comp_stars = [[int(comp_stars[i]), int(comp_stars[i+1])] for i in range(0, len(comp_stars), 2)]
+        comp_stars = re.findall(r"[-+]?(?:\d*\.*\d+)", comp_stars)
```

Comment:
In the non-capturing group, it seems to allow '.<number>'. I was wondering if it was intentional to capture a group starting with a decimal. (Maybe `\d+` or `\d{1,3}`?)

Reply:
I think this is a little safer: '\d+\.?\d*'

Reply:
Done in PR #1248